api/proto/job/job.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+option go_package = "teleport-job-worker/api/proto/job";
+package job;
+
+message StartRequest {
+  job.Command command = 1;
+  job.Resources resource_limits = 2;
+}
+
+message StopRequest {
+  string id = 1;
+}
+
+message QueryRequest {
+  string id = 1;
+}
+
+message OutputRequest {
+  string id = 1;
+}
+
+message Resources {
+  int32 memory = 1;
```

Comment:
Is int32 appropriately sized for this value?

Reply:
It would probably provide enough memory, but I agree, better to give access to all the memory with `uint64`.